runtime/compiler/runtime/MetaData.cpp

```diff
@@ -1480,6 +1493,8 @@ createMethodMetaData(
       Stack alloc map (if exists)
       --------------
       internal pointer map
+      --------------
+      Potential MethodHandle.invokeBasic() call sites
```

Comment:
I'm just comparing this legend with the diagram in [`J9JITExceptionTable.md`](https://github.com/eclipse-openj9/openj9/blob/bc080d2666cd77b2c6c06ca7c5a10552dbfbd29a/doc/compiler/runtime/J9JITExceptionTable.md#High-Level-Overview).  Obviously these aren't new entries that were added by this pull request, but is this legend missing entries for OSR Info and GPU Info?  Similarly, is a new entry needed in `J9JITExceptionTable.md` for the `MethodHandle.invokeBasic() call sites`?

Those changes could be dealt with in a follow-on pull request.  I don't think it would justify holding up this pull request.